include/eve/module/math/regular/impl/gd.hpp

```diff
@@ -13,7 +13,7 @@
 namespace eve::detail
 {
 
-template<floating_real_value T>
+template<value T>
```

Comment:
this means wide<int> is now a valid parameters. Do you need additional requirements ??

Reply:
there is in fact a small pb. The "real" formula is not correct for the whole complex plane so I will put back floating and add a correct complex version for gd and invgd